src/Select/Select.tsx

```diff
@@ -1,107 +1,52 @@
-import React, { FC, ReactElement } from 'react';
-import SelectBase, { components, OptionsType } from 'react-select';
-import { Icon } from '../Icon/Icon';
-import type {
-  Props as SelectBaseProps,
-  IndicatorProps,
-  OptionTypeBase,
-  GroupTypeBase,
-  OptionProps
-} from 'react-select';
-import { styles } from './shared';
-export type {
-  ValueType,
-  ActionMeta,
-  GroupTypeBase,
-  OptionTypeBase
-} from 'react-select';
-
-export type SelectOptionType<OptionType extends OptionTypeBase> =
-  | OptionType
-  | OptionsType<OptionType>;
-
-export interface SelectProps<
-  OptionType extends OptionTypeBase,
-  IsMulti extends boolean,
-  GroupType extends GroupTypeBase<OptionType>
-> extends SelectBaseProps<OptionType, IsMulti, GroupType> {
-  icon?: boolean;
-  clearText?: string;
+import React, { ChangeEvent, ReactElement } from 'react';
+import { Input } from 'reactstrap';
+
+export interface SelectProps {
+  /**
+   * L'identificativo univoco del componente
+   */
+  id: string;
+  /**
+   * Se `true` determina la disabilitazione del campo
+   */
+  disabled?: boolean;
+  /**
+   * L'etichetta del campo
+   */
+  label: string;
+  children: ReactElement<'option'>[];
+  /**
+   * Gestisce il cambio del valore selezionato
+   * @param selectedValue
+   * @returns
+   */
+  handleChange: (selectedValue: string) => void;
```

Comment:
The problem with this is that `SelectHTMLAttributes` already has an `onChange` property with a different signature, which accepts an event instead of a string

Reply:
Actually, it has also a `handleChange`

Reply:
I implemented the aforementioned way to override the signature (using `Omit<K,V>`) but feel free to to suggest better alternatives